specs/result.spec.ts

```diff
@@ -217,4 +217,10 @@ describe('Result', () => {
     done()
   })
 
+  it('should return the value or error', () => {
+    expect(Ok(6).value()).toBe(6)
+
+    expect(Err(new Error("Test Error")).value()).toEqual(new Error("Test Error"))
```

Comment:
Let's just use a string to make sure we're not doing an equality check on a full object which may not give desired effects
```suggestion
    expect(Err("Test Error").value()).toEqual("Test Error")
```

Reply:
Fixed in commit e8f5400f54471f7d1a9d2ebbdbe553aad6fced65